cmd/kubelet/app/options/options.go

```diff
@@ -39,169 +40,96 @@ const (
-		AuthPath:                    util.NewStringFlag("/var/lib/kubelet/kubernetes_auth"), // deprecated
-		CAdvisorPort:                4194,
-		CertDirectory:               "/var/run/kubernetes",
-		CgroupRoot:                  "",
-		ConfigureCBR0:               false,
-		ContainerRuntime:            "docker",
-		CPUCFSQuota:                 false,
-		DockerDaemonContainer:       "/docker-daemon",
-		DockerExecHandlerName:       "native",
-		EventBurst:                  10,
-		EventRecordQPS:              5.0,
-		EnableDebuggingHandlers:     true,
-		EnableServer:                true,
-		FileCheckFrequency:          20 * time.Second,
-		HealthzBindAddress:          net.ParseIP("127.0.0.1"),
-		HealthzPort:                 10248,
-		HostNetworkSources:          kubetypes.AllSource,
-		HostPIDSources:              kubetypes.AllSource,
-		HostIPCSources:              kubetypes.AllSource,
-		HTTPCheckFrequency:          20 * time.Second,
-		ImageGCHighThresholdPercent: 90,
-		ImageGCLowThresholdPercent:  80,
-		KubeConfig:                  util.NewStringFlag("/var/lib/kubelet/kubeconfig"),
-		LowDiskSpaceThresholdMB:     256,
-		MasterServiceNamespace:      api.NamespaceDefault,
-		MaxContainerCount:           100,
-		MaxPerPodContainerCount:     2,
-		MaxOpenFiles:                1000000,
-		MaxPods:                     40,
-		MinimumGCAge:                1 * time.Minute,
-		NetworkPluginDir:            "/usr/libexec/kubernetes/kubelet-plugins/net/exec/",
-		NetworkPluginName:           "",
-		VolumePluginDir:             "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
-		NodeLabels:                  []string{},
-		NodeLabelsFile:              "",
-		NodeStatusUpdateFrequency:   10 * time.Second,
-		OOMScoreAdj:                 qos.KubeletOOMScoreAdj,
-		PodInfraContainerImage:      kubetypes.PodInfraContainerImage,
-		Port:                           ports.KubeletPort,
-		ReadOnlyPort:                   ports.KubeletReadOnlyPort,
-		RegisterNode:                   true, // will be ignored if no apiserver is configured
-		RegisterSchedulable:            true,
-		RegistryBurst:                  10,
-		RegistryPullQPS:                5.0,
-		ResourceContainer:              "/kubelet",
-		RktPath:                        "",
-		RktStage1Image:                 "",
-		RootDirectory:                  defaultRootDir,
-		SerializeImagePulls:            true,
-		StreamingConnectionIdleTimeout: 5 * time.Minute,
-		SyncFrequency:                  1 * time.Minute,
-		SystemContainer:                "",
-		ReconcileCIDR:                  true,
-		KubeAPIQPS:                     5.0,
-		KubeAPIBurst:                   10,
-		ExperimentalFlannelOverlay:     experimentalFlannelOverlay,
-		OutOfDiskTransitionFrequency:   5 * time.Minute,
+		AuthPath:              util.NewStringFlag("/var/lib/kubelet/kubernetes_auth"), // deprecated
+		KubeConfig:            util.NewStringFlag("/var/lib/kubelet/kubeconfig"),
+		NodeLabels:            []string{},
```

Comment:
I think we would want them to be a map[string]string if we represented them in KubeletConfiguration. I've tried to make as direct a mapping from command line args to the KubletConfiguration struct as possible. Stuff that I was unsure of I left out.